Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2924,6 +2924,8 @@ - (void)_hidePasswordView
     self._currentContentView.hidden = NO;
     [_passwordView hide];
     _passwordView = nil;
+
+    _navigationState->didStopRequestingPasswordForQuickLookDocument();
```

Comment:
Right?

```suggestion
#if USE(QUICK_LOOK)
    _navigationState->didStopRequestingPasswordForQuickLookDocument();
#endif
```

Reply:
Wot?

Reply:
I meant "You have to use this suggestion instead, right?"

Reply:
Looks like that's what you pushed in https://github.com/WebKit/WebKit/commit/426c3a65af6c5d0b987c9cf2afab4166fa585e12 :P